docs/decisions/0003-custom-user-model.md

```diff
@@ -0,0 +1,34 @@
+# 0002 Use Custom User Model
+
+- Date: 2024-04-10
+- Author(s): [Chen Zhang][chen]
+- Status: `Active`
+
+## Decision
+
+We will use a custom User model instead of using Django built-in User model.
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏Can we also state here that the custom model will be _based_ on the built in one and provide all the same features?

Reply:
Thanks, this makes lot of sense.